db/schema.rb

```diff
@@ -44,8 +44,11 @@
     t.string   "link_to_github"
     t.datetime "created_at",        null: false
     t.datetime "updated_at",        null: false
+    t.integer  "event_id"
+    t.index ["event_id"], name: "index_workshops_on_event_id", using: :btree
   end
 
   add_foreign_key "events", "sponsors"
   add_foreign_key "events", "workshops"
+  add_foreign_key "workshops", "events"
 end
```

Comment:
Did you make any changes to your database in this pull? 
I don't see any migration file, so please remove these changes from this pull 😄